techniques/aria/ARIA11.html

```diff
@@ -22,6 +22,9 @@
 </li>
          <li>navigation: A collection of links suitable for use when navigating the document or related documents.
 </li>
+         <li>region: A generic region of content to which, for an author-specific purpose, it is important for users to be able to navigate. In HTML5, `section` with an accessible label maps to the region role. The region role needs an accessible label to be included as a landmark for assistive technology.
```

Comment:
The current version of the [ARIA11 technique](https://w3c.github.io/wcag/techniques/aria/ARIA11) only lists "HTML" or "HTML4". So I don't think it's necessary to say "HTML5" here.

```suggestion
         <li>region: A generic region of content to which, for an author-specific purpose, it is important for users to be able to navigate. In HTML, `section` with an accessible label maps to the region role. The region role needs an accessible label to be included as a landmark for assistive technology.
```